scripts/setup-parachain-docker.ts

```diff
@@ -1,4 +1,5 @@
 import shell from "shelljs";
+import minimist from "minimist";
```

Comment:
so far we've been using `yargs`, we should stick to one CLI lib for consistency across repos. Is minimist better? 

Example usage of yargs: https://github.com/interlay/bridge-bot/blob/ed7d42f893520d8fe7d806e12c9d9c0a781a6025/src/index.ts#L19